infrastructure/cdn-in-a-box/traffic_stats/Dockerfile

```diff
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+############################################################
+# Dockerfile to build Traffic Stats container images
+# Based on CentOS
+############################################################
+
+FROM centos:7
+
+# Default values for RPM -- override with `docker build --build-arg RPM=...'
+ARG TRAFFIC_TS_RPM=traffic_stats/traffic_stats.rpm
+ADD $TRAFFIC_TS_RPM /
+
+RUN yum install -y epel-release && \
+    yum install -y /$(basename $TRAFFIC_TS_RPM) \
+        jq \
+        nmap-ncat \
+        iproute \
+        net-tools \
+        gettext \
+        bind-utils \
+        openssl \
+        initscripts && \
```

Comment:
Does this really need, `iproute` and `initscripts`? I'm pretty sure the others are all boilerplate (`bind-utils` is the CentOS equivalent of `dnsutils`, right?), but those seem unique to me.